wazo_call_logd/plugins/cdr/resource.py

```diff
@@ -84,62 +94,64 @@ def _output_csv(data, code, http_headers=None):
     return response
 
 
-class CDRResource(AuthResource):
-
+class CDRAuthResource(AuthResource):
     representations = {'text/csv; charset=utf-8': _output_csv}
 
     def __init__(self, cdr_service):
+        super().__init__()
```

Comment:
Only for styling/best practices?

Reply:
Just best practices, I don't want to known if `AuthResource` have `__init__` implemented or not. Maybe today it haven't, who known later. Python always call the object() one for us, but it's only magisk, in some other oom languages it doesn't.

When there is too much inheritances, you don't want to check all parents, grand parents, ... class to see if someone in the tree have a custom __init__. Always adding it always work, while missing one can introduce nasty bug (like code looks like to work, but one var is not initialized as expected).